projects/Mallard/src/components/spinner.tsx

```diff
@@ -72,7 +72,7 @@ const Spinner = () => {
     ])
     useEffect(() => {
         Animated.parallel(jumps.map((j, i) => animateJumps(j, i))).start()
-    }, [])
+    }, [jumps])
```

Comment:
this needs an ignore line instead, triggering this animation more than once in the components life is gonna look janky

Reply:
Also use-image-paths?